jsk_2016_01_baxter_apc/scripts/create_udev_rules

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Yes, the same as rosserial rules.
So I should just write rules in README.
I'll fix this.